files/lib/data/faq/Question.class.php

```diff
@@ -7,11 +7,14 @@
 use wcf\data\faq\category\FaqCategory;
 use wcf\data\user\User;
 use wcf\data\DatabaseObject;
+use wcf\data\search\ISearchResultObject;
+use wcf\page\FaqQuestionPage;
 use wcf\system\html\output\HtmlOutputProcessor;
 use wcf\system\request\IRouteController;
+use wcf\system\request\LinkHandler;
 use wcf\system\WCF;
 
-class Question extends DatabaseObject implements IRouteController
+class Question extends DatabaseObject implements IRouteController, ISearchResultObject
```

Comment:
As you can see in the search result list, the page icon is displayed by default because (for obvious reasons) there's no user profile associated with the question object:
![question search result with default page icon](https://user-images.githubusercontent.com/12609161/163674131-7298c6c1-6804-4137-a918-8be5a4672596.png)

We would be able to replace the default page icon with a question mark icon (just my suggestion that seems most appropriate, but I'm open to other suggestions) that would look like this:
![question search result with question mark icon](https://user-images.githubusercontent.com/12609161/163674191-5d7432da-416d-471a-9fa6-0bc0edf3ece7.png)

To do this, simply implement the interface [`ICustomIconSearchResultObject`](https://github.com/WoltLab/WCF/blob/master/wcfsetup/install/files/lib/data/search/ICustomIconSearchResultObject.class.php) instead of `ISearchResultObject`, implement the interface's `getCustomSearchResultIcon` function and return `fa-question` (or whatever you like).